src/RdKafka/Topic.cs

```diff
@@ -56,22 +60,36 @@ public void Dispose()
 
         public string Name => handle.GetName();
 
+        public void Produce(IDeliveryHandler deliveryHandler, byte[] payload, byte[] key = null, Int32 partition = RD_KAFKA_PARTITION_UA)
```

Comment:
Yes, i would like to move it to the end but then i need to remove the defaults for the preceding args. What do you think?